cluster-autoscaler/cloudprovider/aws/aws_cloud_provider_test.go

```diff
@@ -27,6 +25,7 @@ import (
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/aws/aws-sdk-go/aws"
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider/aws/aws-sdk-go/service/autoscaling"
 	"k8s.io/autoscaler/cluster-autoscaler/config"
+	"testing"
```

Comment:
Why did the import order change here?

Reply:
Verify script was failing. This change is a result of `hack/update-gofmt.sh` script.